scripts/set_kokkos_env.sh

```diff
@@ -0,0 +1,12 @@
+# set nvcc as the underlying C++ compiler for Open MPI
+export OMPI_CXX=$TRILINOS_DIR/packages/kokkos/config/nvcc_wrapper
+
+# set nvcc as the underlying C++ compiler for MPICH
+export MPICH_CXX=$TRILINOS_DIR/packages/kokkos/config/nvcc_wrapper
+
+# set the original C++ compiler as the underlying compiler for the nvcc
+# wrapper
+export NVCC_WRAPPER_DEFAULT_COMPILER=$PATH_TO_CXX_COMPILER
+
+# set all cuda calls to blocking launch
+export CUDA_LAUNCH_BLOCKING=1
```

Comment:
We may want to call fence(). If we force people to use CUDA_LAUNCH_BLOCKING then apps using DTK that don't use Kokkos will be forced to use blocking kernels as well. I see this as a major restriction.